extensions/search-router/README.md

```diff
@@ -47,6 +48,7 @@ To add new search engines to the extension:
 
 ## Credits 🙏
 
-- [DuckDuckGo !Bangs](https://duckduckgo.com/bangs) - The original implementation of the bangs concept 🦆
+- [Kagi Search Bangs](https://help.kagi.com/kagi/features/bangs.html) - The implementation of the bangs concept 🔍
+- [Kagi Bangs Repository](https://github.com/kagisearch/bangs) - The official repository of Kagi Search bangs 📚
```

Comment:
I think we should still mention DuckDuckGo Bangs in the README, although I agree Kagi should also be mentioned as you added

Reply:
thanks for comment!
updated in https://github.com/raycast/extensions/pull/17996/commits/8bf27afd23482205ea17c2efc76c3c6c0ea23129